doc/themes/the-things-stack/layouts/index.html

```diff
@@ -1,15 +1,134 @@
 {{ define "main" }}
 
-<div class="section">
-  <h1 class="title is-size-2 title-intro has-vertical-space">The Things Stack Documentation</h1>
-  <div class="content-intro">
-    <p>
-      The Things Stack is a robust, yet flexible LoRaWAN® Network Server that caters to the needs of demanding LoRaWAN deployments, from covering the essentials to advanced security configurations and device life cycle management.
-    </p>
-  </div>
+<div>
+
+  <section class="section-doc-home-page-hero ttui-container grid pt-ls-xl pb-ls-xl">
+
+    <div class="item-5 md:item-12 grid sm:mb-ls-l">
+      <div class="mb-ls-l item-12 md:item-6 sm:item-8 xs:item-12">
+        <p class="bg-tts-primary-main d-inline-block c-white ttui-body p-sides-cs-s pt-cs-xxs pb-cs-xxs br-xxl mb-cs-m">v{{- .Site.Params.version -}}</p>
+        <h1 class="mb-cs-s c-interactive-03">The Things Stack Documentation</h1>
+        <p class="ttui-body fs-l mb-cs-xl">The Things Stack is a robust, yet flexible LoRaWAN® Network Server that caters to the needs of demanding LoRaWAN deployments, from covering the essentials to advanced security configurations and device life cycle management.</p>
+        <div class="d-flex gap-cs-m">
+          <a href="https://www.thethingsindustries.com/stack/plans">
+            <button class="ttui-btn ttui-btn--primary">Choose your plan</button>
+          </a>
+          <a href="https://www.thethingsindustries.com/contact">
+            <button class="ttui-btn ttui-btn--secondary">Talk to an expert -></button>
+          </a>
+        </div>
+      </div>
+      <div class="item-12 md:item-6 sm:item-12">
+        <p class="ttui-body fs-l fw-bolder mb-cs-xl">Try it out</p>
+        <div class="container-whats-new-list flex-column b-line-02 br-l">
+          <a class="ttui-link ttui-link--highlight p-cs-m" href="https://www.thethingsindustries.com/docs/getting-started/lorawan-basics">
```

Comment:
We need templated links here.

Even better is looping over the cards like below. I think this is what @pierrephz meant in his comment that this should be fixed.